co-simulation/fed/mosaic-infrastructure/src/main/java/org/eclipse/mosaic/fed/infrastructure/ambassador/InfrastructureInstance.java

```diff
@@ -156,7 +223,7 @@ public void setTimeSyncPort(int timeSyncPort) {
      *                     methods
      */
     public void bind() throws IOException {
-        rxMsgsSocket = new DatagramSocket();
+        socket = new DatagramSocket();
```

Comment:
renamed to `socket` since this Datagramsocket is used now to transmit Time Sync, V2X and Simulated Interactions. RxMsgsSocket indicates it relates the only to the rxMessagePort used for V2X messages.

Reply:
This new DatagramSocket is to create a new UDP  socket and bind to any available port on the host machine according to: https://docs.oracle.com/javase/8/docs/api/java/net/DatagramSocket.html.
However, it seems that we want to create a Client side UDP socket and connect to remote infrastructure instance with IP (targetAddress) and the ports specified (rxMessagePort, timeSyncPort,simulatedInteractionPort ). 
This socket here does not tell which port and IP to send each type of message (V2x, TimeSync, SimulatedInteraction)?

Reply:
The bind() gives the vibe that we are creating a UDP server socket and trying to bind it to a particular port. But this infrastructure instance in fact acts as a series of UDP clients and send packet to remote infrastructure. Maybe we can just call this `socket = new DatagramSocket` inside the infrastructure instance initialization.

Reply:
This is not really part of the scope for this PR but I can look into a better naming convention for this method. It is a single client though, just different ports which can be assigned at the datagrampacket level not the socket level.

Reply:
Maybe `connect` is a better name or something along those lines? the primary reason - if I recall correctly - for having a separate method to init the DatagramSocket was to avoid a possible failure in the constructor (e.g. the ports are already in use) which is just a weird state to have. Not strongly attached to this implementation though.